packages/appsync-modelgen-plugin/src/visitors/appsync-model-introspection-visitor.ts

```diff
@@ -43,8 +38,8 @@ export class AppSyncModelIntrospectionVisitor<
     );
 
     const modelIntrosepctionSchema = this.generateModelIntrospectionSchema();
-    if (!this.schemaValidator(modelIntrosepctionSchema)) {
-      throw new Error(`Data did not validate against the supplied schema. Underlying errors were ${JSON.stringify(this.schemaValidator.errors)}`);
+    if (!validateJsonFile(modelIntrosepctionSchema)) {
```

Comment:
If we don't already have them, please add both positive & negative tests that exercise this validation

Reply:
Thanks. I've added some basic test cases for the standalone validator. The complex cases are already covered by the MIS visitor tests.